puppet/modules/quickstack/manifests/pacemaker/redis.pp

```diff
@@ -0,0 +1,40 @@
+class quickstack::pacemaker::redis(
+  $bind_host             = '127.0.0.1',
+  $port                  = '6379',
+  $slaveof               = undef,
+) {
+
+  $redis_group = map_params("redis_group")
+
+  class {'::quickstack::firewall::redis':
+    ports => [$port],
+  }
+
+  class {'::quickstack::db::redis':
+    bind_host => $bind_host,
+    port      => $port,
+    slaveof   => $slaveof,
+  }
+
+  quickstack::pacemaker::resource::generic {'redis':
+    clone_opts    => "interleave=true",
+    operation_opts => "wait_last_known_master=true notify=true ordered=true"
+    resource_name => "redis",
+    resource_params => "--master meta",
```

Comment:
This was from the scenario list of commands. But i think thats just referencing the mode see https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Configuring_the_Red_Hat_High_Availability_Add-On_with_Pacemaker/s1-multistateresource-HAAR.html